WcaOnRails/app/controllers/static_pages_controller.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
-include ActionView::Helpers::AssetTagHelper
 class StaticPagesController < ApplicationController
+  include ActionView::Helpers::AssetTagHelper
```

Comment:
@jfly any idea why you included it in the very first Rails commit (c5ddd14067bee70172ccab11767313d25be51de0)? Quite a long time ago ^^ It doesn't seem necessary to me.

Reply:
No idea, sorry! If removing it works, let's get rid of it!